browser/importer/brave_profile_writer.cc

```diff
@@ -300,9 +300,13 @@ void BraveProfileWriter::UpdateLedger(const BraveLedger& ledger) {
   }
 
   ledger_ = BraveLedger(ledger);
+  rewards_service_->IsWalletCreated(
```

Comment:
this just came up recently I think with @ryanml. We should be checking `rewards_service_->is_enabled()` - but this doesn't seem related to this PR?

Reply:
I see, it's because IsWalletCreated now has a callback. In either case we should always check to ensure that rewards service is enabled before calling any methods on it